cmd/cosign/cli/verify/verify_blob_attestation_test.go

```diff
@@ -118,3 +118,78 @@ func TestVerifyBlobAttestation(t *testing.T) {
 		})
 	}
 }
+
+func TestVerifyBlobAttestationPolicy(t *testing.T) {
```

Comment:
Would it make sense to include a rego policy in here too?  Probably something like

```
package signature
allow {
  input.predicateType == "https://slsa.dev/provenance/v0.2"
}
```

(borrowed from rego_test.go)

Reply:
yes, i'll do one w/ rego and one with both rego + cue. you caught me that I didn't know rego :P

Reply:
Ha, me neither. :)

Reply:
done